### DISCUSSION TITLE
Enable auto-merge

### DISCUSSION THREAD

**Discussion by j0k3r** [serverless-heaven/serverless-webpack#1557] [Team Posts]
Hi @HyperBrain,

Can you enable _Allow auto-merge_ in the serverless-webpack repository settings?
@vicary add that feature to merge automatically minor deps version: https://github.com/serverless-heaven/serverless-webpack/pull/1008

Thanks!

**Comment by HyperBrain** [comment]
Hi,

yes, will do soon :) sorry for the delayed response.

- Frank

**Comment by j0k3r** [comment]
Hi @HyperBrain,

Happy new year.
Can you find time to unblock us?
- allow _Allow auto-merge_ in the repository settings
- remove the requirement for the `coverage/coveralls` build

Thanks!

**Comment by HyperBrain** [comment]
auto-merge has been enabled now
